docs/framework/common-client-technologies/how-to-configure-client-application-services.md

```diff
@@ -137,7 +137,7 @@ This topic describes how to use the Visual Studio **Project Designer** to enable
 ## See Also  
  [Client Application Services](../../../docs/framework/common-client-technologies/client-application-services.md)  
  [Client Application Services Overview](../../../docs/framework/common-client-technologies/client-application-services-overview.md)  
- [Services Page, Project Designer](https://msdn.microsoft.com/library/bb398109)  
+ [Services Page, Project Designer](/visualstudio/ide/reference/signing-page-project-designer)  
```

Comment:
signing --> services

Reply:
we should check if I made the same mistake when replacing links pointing to the "Signing Page, Project Designer"

Reply:
I didn't spot others, but I'll let you double check @rpetrusha